app/scripts/views/tooltip.js

```diff
@@ -27,6 +27,14 @@ define([
       this.dismissible  = options.dismissible || false;
       this.extraClassNames = options.extraClassNames || '';
 
+      // Event codes for the keyboard keys.
+      this.DOWN_ARROW_KEY_CODE = 40;
```

Comment:
@shane-tomlinson Do you want me to do that next? I can create that file now and start cleaning the hard coded 13.